src/spec/page-objects/settings-page.js

```diff
@@ -81,5 +85,19 @@ class SettingsPage {
       return val[0].click();
     });
   }
+  clickFirstDkimInformationButton(){
+    return this._dKimInformationButtons.then(function(val){
+      return val[0].click();
+    });
+  }
+  getdKimDomainSelected(){
+    return this._dKimDomainSelected.getText();
+  }
+  getdKimDomainSelector(){
```

Comment:
These two fields seems to be part of another page, I am not sure if `settings-page` is a good place for them.